packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -86,14 +90,14 @@ const subscribe = ({ onFlagsStateChange, onStatusStateChange }) =>
   });
 
 const configure = ({
-  clientSideId,
+  authorizationKey,
```

Comment:
I felt we should rename this to what it actually is in a splitio context.

Reply:
Yup, I named like this first but I wanted to expose the same API as LD. `authorizationKey` fit better.

Reply:
Yeah I think it can really and should depend on the adapter. Also so that it reads nice on the configuration side

```js
<ConfigureFlopflip adapter={splitioAdapter} adapterArgs={{ authorizationKey }} />
```

Note that the `adapterArgs` shoudl always be a memoized thing so the reference does not change on each call to render.